packages/amplify_analytics_pinpoint/lib/method_channel_amplify.dart

```diff
@@ -74,7 +74,10 @@ class AmplifyAnalyticsPinpointMethodChannel extends AmplifyAnalyticsPinpoint {
   }
 
   @override
-  Future<void> unregisterGlobalProperties({List<String> propertyNames}) async {
+  Future<void> unregisterGlobalProperties({List<String>? propertyNames}) async {
+    if (propertyNames == null) {
+      propertyNames = <String>[];
+    }
```

Comment:
Think we can avoid this check by providing a default:
```dart
Future<void> unregisterGlobalProperties({List<String>? propertyNames = const []}) async {
```
We can also safely cast as non-null after that if necessary

Reply:
sounds good, updated the code.